app/webpacker/components/StaticPages/Logo.jsx

```diff
@@ -0,0 +1,169 @@
+        />
+      </p>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.paragraph.see_below`}
+        />
+      </p>
+      <h3>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.title`}
+        />
+      </h3>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.paragraph`}
+        />
+      </p>
+      <h4>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.primary.title`}
+        />
+      </h4>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.primary.paragraph`}
+        />
+      </p>
+      <div style={sideBySideStyle}>
+        <img src={lockupPrimary} alt="Primary logo" style={lockupStyle} />
+        <img src={lockupNegative} alt="Primary logo" style={lockupStyleDark} />
+      </div>
+      <h4>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.mono.title`}
+        />
+      </h4>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.color.mono.paragraph`}
+        />
+      </p>
+      <div style={sideBySideStyle}>
+        <img src={lockupMonoBlack} alt="Primary logo" style={lockupStyle} />
+        <img src={lockupMonoWhite} alt="Primary logo" style={lockupStyleDark} />
+      </div>
+      <h3>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.logo_only.title`}
+        />
+      </h3>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.usage_guidelines.logo_only.paragraph`}
+        />
+      </p>
+      <div style={sideBySideStyle}>
+        <img src={logoOnlyPrimary} alt="Primary logo" style={logoOnlyStyle} />
+        <img
+          src={logoOnlyNegative}
+          alt="Primary logo"
+          style={logoOnlyStyleDark}
+        />
+        <img src={logoOnlyMonoBlack} alt="Primary logo" style={logoOnlyStyle} />
+        <img
+          src={logoOnlyMonoWhite}
+          alt="Primary logo"
+          style={logoOnlyStyleDark}
+        />
+      </div>
+      <h2>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.download_logo_assets.title`}
+        />
+      </h2>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey={`logo.headings.download_logo_assets.paragraph`}
+        />
+      </p>
+      <div>
+        <label>
+          <input
+            type="checkbox"
+            name="checkbox"
+            id="checkbox_id"
+            value={acceptedGuidelines}
+            onClick={() => setAcceptedGuidelines(!acceptedGuidelines)}
+            style={{ marginRight: "10px" }}
+          />
+          <I18nHTMLTranslate
+            i18nKey={`logo.headings.download_logo_assets.accept_terms_and_conditions`}
+          />
+        </label>
+      </div>
+      <button
+        disabled={!acceptedGuidelines}
+        onClick={downloadFile}
```

Comment:
Handling a download using `onClick` is a bit unorthodox. SemanticUI has this neat `as` property, try and see if the following will work:
```jsx
<Button
  disabled={!acceptedGuidelines}
  positive
  as="a"
  href="/whatever/link/you/want"
>
  Click me please!
</Button>
```